spec/index.bs

```diff
@@ -1762,7 +1790,7 @@ GPUBuffer includes GPUObjectBase;
         Issue(gpuweb/gpuweb#605): Specify {{GPUBuffer/[[mapping]]}} in term of `DataBlock` similarly
         to `AllocateArrayBuffer`?
 
-    : <dfn>\[[mapping_range]]</dfn> of type [=list=]&lt;[=Number=]&gt; or `null`.
+    : <dfn>\[[mapping_range]]</dfn> of type [=list=]&lt;{{unsigned long long}}&gt; or `null`.
```

Comment:
Unrelated?

Reply:
Yes, just noticed it because I was adding a cross ref to css and noticed we accidentally had one here ("Number" autolinks to css, incorrectly)